### ISSUE TITLE
Git pathspec issue

### ISSUE THREAD

**Issue by wweaver** [datasift/gitflow#94]
When running `git hf update` I get a warning about git no longer supporting relative pathspecs. I keep up to date with the latest version of git via homebrew so I assume when git decides to remove the ability hubflow will stop working.

```
warning: empty strings as pathspecs will be made invalid in upcoming releases. please use . instead if you meant to match all paths
error: pathspec '' did not match any file(s) known to git.
Unable to checkout branch ''
```

I haven't taken the time to look to see if this is a simple fix or not, but wanted to report it as a bug at least.

**Comment by stuartherbert** [comment]
Thank you for that report. It's been fixed in v1.5.3.

I'd done a stupid thing with a stack operation inside `hubflow-common`.

**Comment by wweaver** [comment]
Thanks. That seems to have fixed it. 👍
